src/main/webapp/WEB-INF/cars/cars.jsp

```diff
@@ -0,0 +1,38 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: valdemar
+  Date: 26.06.2023
+  Time: 18:33
+  To change this template use File | Settings | File Templates.
+--%>

```

Comment:
As stated in common mistakes, it is necessary to remove autogenerated comments from JSPs.